com.unity.netcode.gameobjects/CHANGELOG.md

```diff
@@ -39,7 +45,6 @@ Additional documentation and release notes are available at [Multiplayer Documen
 
 ### Fixed
 
-- Fixed not sending all NetworkVariables to all clients when a client connects to a server. (#1987)
```

Comment:
This change actually was not part of 1.0.1. It was made after 1.0.1 was published. So it should be in the [Unreleased] section.